helm/alfresco-content-services/values.yaml

```diff
@@ -274,6 +275,9 @@ alfresco-transform-service:
       data:
         mountPath: "/tmp/Alfresco"
         subPath: "alfresco-content-services/filestore-data"
+    strategy:
+      # -- Strategy must be set to Recreate when persistence use a `ReadWriteOnce` storageClass
+      type: Recreate
```

Comment:
not sure this is the best place where to set this given that it really depends on the `accessMode` of the `storageClass`

Reply:
I looked up postgres and redis charts they use the same approach, strategy is defined outside of the `storageClass`. Still not sure if this the best practice but looks good

Reply:
I would have set it her eas well. Though I would have added helm-docs annotations with the opposite thinking: mentioning what to do to set more than one replica instead of why it is set to one.